src/bscPlugin/validation/BrsFileValidator.ts

```diff
@@ -51,13 +53,15 @@ export class BrsFileValidator {
                 this.validateEnumDeclaration(node);
 
                 //register this enum declaration
-                node.parent.getSymbolTable()?.addSymbol(node.tokens.name.text, node.tokens.name.range, DynamicType.instance);
+                // eslint-disable-next-line no-bitwise
+                node.parent.getSymbolTable()?.addSymbol(node.tokens.name.text, node.tokens.name.range, DynamicType.instance, SymbolTypeFlags.typetime | SymbolTypeFlags.runtime);
```

Comment:
members at runtime, not enum itself -- change to typetime only

Reply:
NOTE - this needed to be runtime, so that `enum.value` expressions won't be caught